packages/server/graphql/intranetSchema/mutations/updateOAuthRefreshTokens.ts

```diff
@@ -0,0 +1,27 @@
+import {GraphQLInt} from 'graphql'
+import {requireSU} from '../../../utils/authorization'
+import {GQLContext} from '../../graphql'
+import getAtlassianAuthsToUpdate from '../../../postgres/queries/getAtlassianAuthsToUpdate'
+import {Threshold} from '~/types/constEnums'
+import {isNotNull} from '../../../utils/predicates'
+
+const updateOAuthRefreshTokens = {
+  type: GraphQLInt,
+  description: `Updates Atlassian OAuth tokens that haven't been updated for more than 14 days`,
+  args: {},
+  resolve: async (_source, {}, {authToken, dataLoader}: GQLContext) => {
+    //AUTH
+    requireSU(authToken)
+
+    // RESOLUTION
+    const olderThan14DaysThreshold = new Date(Date.now() - Threshold.OAUTH_REFRESH)
+    const atlassianAuthsToUpdate = await getAtlassianAuthsToUpdate(olderThan14DaysThreshold)
+    const updatedAtlassianAuths = (
+      await dataLoader.get('freshAtlassianAuth').loadMany(atlassianAuthsToUpdate)
+    ).filter(isNotNull)
```

Comment:
let's also apply `errorFilter`
the only way we'd get an `instanceof Error` is if our batchFn throws an error that we don't know is an error.
e.g. if we added a JSON.parse() somewhere in the batchFn & it threw

Reply:
Good to know! Added 👍